docs/EXTENDED-ARCHIVES.md

```diff
@@ -0,0 +1,428 @@
+Etcher Extended Archives
+========================
+
+Etcher extended archives are archive files including an image and a special
+metadata directory including information that describes the image, device type,
+and vendor in detail, enabling Etcher to make use of such information to
+deliver advanced features and a more pleasant experience to the user.
+
+Entities
+--------
+
+### Vendor
+
+A vendor represents a manufacturer of a device type and potential publisher.
+
+*Some examples of image vendors are: Beagleboard, Raspberry Pi Foundation,
+Intel, and ODroid.*
+
+### Publisher
+
+A publisher represents an entity that maintains and publishes official images.
+
+*Some examples of image publishers are: Tizen, RetroPie, Resin.io, and CoreOS.*
+
+### Device
+
+A device represents a specific device type manufactured by a vendor.
+
+*Some examples of devices are: Beaglebone Black, Beaglebone Green, Intel
+Edison, and Raspberry Pi 3.*
+
+### Client
+
+A client represents the program making use of the information about the other
```

Comment:
We use the word "client" in various parts of the document, so I though describing precisely what we mean by that can clarify things. What do you think? We can move it to a section called "glossary" or something like that if the word "entity" is the confusing part.